indico/modules/core/controllers.py

```diff
@@ -138,6 +139,8 @@
         if update_user and session.user:
             session.user.settings.set('force_timezone', mode != 'local')
 
+        if redirect_url:
+            return redirect(redirect_url, 303)
```

Comment:
@ThiefMaster This is the issue that we talked about before. If my memory serves me, you said it was fine, but let me know if we should address it. I don't have any fantastic ideas, though.

Reply:
Since it's a POST request I think it's very hard to abuse it as an open redirector. However, assuming this is always going to be a relative URL I recommend a check like this just in case:

```python
if redirect_url.startswith('//') or not redirect_url.startswith('/'):
    redirect_url = '/'
```

I think this prevents any non-relative URLs (since you cannot use `//evil.com/...` nor `https://evil.com`).